### PR TITLE
Syntax highlighting

### PR DESCRIPTION
**Pull request by skade**
Fixup of #606

### CODE REVIEW

File: templates/components/layout.hbs [hunk 1]
```diff
@@ -42,5 +44,9 @@
         {{~> page}}
     </div>
     {{> components/footer}}
+    <script src="/static/scripts/highlight.pack.js"></script>
+    <script>
+        hljs.initHighlightingOnLoad();
+    </script>
```
Comment (pietroalbini) [inline templates/components/layout.hbs]: Can we avoid the inline script? I'm going to try adding a Content Security Policy soon and inline scripts are pretty bad for it.

File: templates/components/layout.hbs [hunk 2]
```diff
@@ -25,6 +25,8 @@
     <link rel="stylesheet" href="/static/styles/vendor.css"/>
     <link rel="stylesheet" href="/static/styles/fonts.css"/>
     <link rel="stylesheet" href="/static/styles/app.css"/>
+    <link rel="stylesheet" href="/static/styles/syntax-highlight/a11y-light2.css"/>
+    <!-- <link rel="stylesheet" href="/static/styles/syntax-highlight/theme.css"/> -->
```
Comment (pietroalbini) [inline templates/components/layout.hbs]: Why is this line commented?

File: src/styles/app.scss [hunk 3]
```diff
@@ -7,7 +7,9 @@ $gray: #454C52;
 $red: #C14566;
 $green: #398277;
 $purple: #403D58;
-$yellow: #FFD45E;
+$yellow: #FECD2F;
+$a11ybg: #fefefe;
```
Comment (pietroalbini) [inline src/styles/app.scss]: Nit: capitalize the hex?

**Comment by skade** [comment]
I agree that it's jarring, but a little less jarring as before. Basically, we'd need a theme per background :/.

**Comment by danilaml** [comment]
Could this be applied to the https://blog.rust-lang.org/ in the meantime? It shouldn't have those background issues, AFAIU.

**Comment by Manishearth** [comment]
Needs rebase

**Comment by skade** [comment]
@Manishearth After discussions with @pietroalbini and madmalik, I'd close this and wait for a new implementation. This has too many problems.

**Pull request closed without merging**